tests/integration/local/invoke/test_invoke_cdk_templates_with_function_id.py

```diff
@@ -63,3 +63,70 @@ def test_invoke_function_with_function_id(self, logical_id, function_name, funct
 
             self.assertEqual(return_code, 0)
             self.assertEqual(response, expected_response)
+
+
+class TestCDKSynthesizedTemplatesNestedFunctionIdentifies(InvokeIntegBase):
+
+    template = Path("cdk/nested_templates/cdk_function_id_parent_template.yaml")
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Since these aren't image functions and they don't need to be built before invoking, I don't think you need the `setUpClass` or `tearDown` methods.

Reply:
Thanks, updated this.